File: Beginner/1038.Snack.go
// https://www.beecrowd.com.br/judge/en/problems/view/1038

package main
import (
"fmt"
)
var price float32 = 0.00

func main() {
	var code, quantity int32

	fmt.Scanln(&code, &quantity)

	switch(code){
		case 1:
			price = (4.00 * float32(quantity))
		case 2:
			price = (4.50 * float32(quantity))
		case 3:
			price = (5.00 * float32(quantity))
		case 4:
			price = (2.00 * float32(quantity))
		case 5:
			price = (1.50 * float32(quantity))
	}
	fmt.Printf("Total: R$ %.2f\n", price)
}

File: Beginner/cpp/1038.Snack.cpp
// https://www.beecrowd.com.br/judge/en/problems/view/1038

#include <bits/stdc++.h>
using namespace std;

int main() {
 
    int code, quantity;
    cin >> code >> quantity;
    
    float price = 0.00;
    switch(code){
        case 1:
            price = (float) (4.00 * quantity);
            break;
        case 2:
            price = (float) (4.50 * quantity);
            break;
        case 3:
            price = (float) (5.00 * quantity);
            break;
        case 4:
            price = (float) (2.00 * quantity);
            break;
        case 5:
            price = (float) (1.50 * quantity);
            break;
        default:
            cout << "Invalid Input\n";
    }
    printf("Total: R$ %.2f\n",price);
 
    return 0;
}